src/utils/dom.js

```diff
@@ -253,3 +262,23 @@ export const getTabables = (el = document) =>
   selectAll(TABABLE_SELECTOR, el)
     .filter(isVisible)
     .filter(i => i.tabIndex > -1 && !i.disabled)
+
+// Attempt to focus an element, and return `true` if successful
+export const attemptFocus = (el, options = {}) => {
+  if (isElement(el) && el.focus) {
+    try {
+      el.focus(options)
+    } catch {}
+  }
+  return isActiveElement(el)
+}
+
+// Attempt to blur an element, and return `true` if successful
+export const attemptBlur = el => {
+  if (isElement(el) && el.blur) {
+    try {
+      el.blur()
+    } catch {}
+  }
+  return !isActiveElement(el)
```

Comment:
Similar here, we can remove the outer `if` check, as the `try/catch` will handle attempts to blur a non-blurable element